### DISCUSSION TITLE
Git on Local Repos!

### DISCUSSION THREAD

**Discussion by arimakidd** [community/community#23922] [New to GitHub]
<p>I use Github and it works fine.  But it looks like local repos behave very differently.  I created a git repo by ‘cd’ into a folder and ran</p>
<pre><code class="lang-auto">git init
</code></pre>
<p>I created 2 files, add and commit.  Good so far.  Then I decided to clone the local repo into another folder.</p>
<pre><code class="lang-auto">git clone &lt;folderONe&gt; &lt;folderTwo&gt;
</code></pre>
<p>Made some change in folderTwo.  Then I wanted to push these changes back to the original folder which was folderOne.  So I ran a</p>
<pre><code class="lang-auto">git push origin master
</code></pre>
<p>Got the following errors:<br>
<img src="https://user-images.githubusercontent.com/46945387/181080034-f0b87a84-cb02-4bf1-b630-2d3213a3b2ab.png" alt="image" data-base62-sha1="8FPtP0TatnPE3EAup0i0Ubcy9N5" width="644" height="434"><br>
what should I do?  All I want to do is push the changes from folderTwo back to the original repo which is in folderOne.  This works fine when cloning from repos created on Github but creating repos locally seem to work differently.</p>

**Comment by arimakidd** [comment, marked as answer]
<p>I got the answer for this issue.  I edited the config file of the real ‘master’ reop, folderOne with the following:</p>
<pre><code class="lang-auto">[receive]
denyCurrentBranch = false

</code></pre>
<p>Once I did that I was able to go to folderTwo and execute a</p>
<pre><code class="lang-auto">push origin master
</code></pre>
<p>Got the info from this site:<br>
<a href="https://www.wikitechy.com/errors-and-fixes/sql/git-push-error-remote-rejected-master-master-branch-is-currently-checked-out" rel="noopener nofollow ugc">https://www.wikitechy.com/errors-and-fixes/sql/git-push-error-remote-rejected-master-master-branch-is-currently-checked-out</a></p>
<p>SOLVED</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="arimakidd" data-post="2" data-topic="195588">
<div class="title">
<div class="quote-controls"></div>
 arimakidd:</div>
<blockquote>
<p>I got the answer for this issue.</p>
</blockquote>
</aside>
<p>Thank you for posting the solution you’ve found, others will benefit from it.</p>
<p>It’s actually an interesting use case — I never explored this aspect of using Git locally.</p>
